src/main/java/org/apache/commons/net/imap/IMAPCommand.java

```diff
@@ -84,7 +84,7 @@ public static final String getCommand(final IMAPCommand command) {
     }
 
     /**
-     * Get the IMAP protocol string command for this command
+     * Returns the IMAP protocol string command for this command
```

Comment:
I usually normalize Javadoc comments for getters to "Gets..." in all of Commons, so let's keep going in that direction IMO.

Reply:
Got it, will use "Gets ..." in the future 😀